ai_productivity_analyzer.py

```
@@ -15,58 +15,157 @@
+                 top_days: int = DEFAULT_TOP_DAYS_LIMIT):
         """Initialize the analyzer with validated parameters."""
         try:
-            self.ai_start_date = datetime.strptime(ai_start_date, "%Y-%m-%d")
+            # Parse date with timezone awareness
+            naive_date = datetime.strptime(ai_start_date, "%Y-%m-%d")
+            if timezone_str == "UTC":
+                self.ai_start_date = naive_date.replace(tzinfo=timezone.utc)
+            else:
+                # For other timezones, would need pytz or zoneinfo
+                self.ai_start_date = naive_date.replace(tzinfo=timezone.utc)
+                print(f"Note: Using UTC timezone. Custom timezone support coming soon.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)